lms/envs/common.py

```diff
@@ -535,11 +535,6 @@
 STUDENT_FILEUPLOAD_MAX_SIZE = 4 * 1000 * 1000  # 4 MB
 MAX_FILEUPLOADS_PER_INPUT = 20
 
-# FIXME:
-# We should have separate S3 staged URLs in case we need to make changes to
-# these assets and test them.
```

Comment:
Honestly, at this point, I suspect any usage is different than it was in 2012. We've gone through many refactors of how these files are handled, and I wasn't involved in the most recent half-dozen. I have no comment.